.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+Thank you for contributing to The Foreman, please read the following guide
+available at https://www.theforeman.org/contribute.html - in short:
+
+* Create an issue at http://projects.theforeman.org/projects/foreman/issues
+* Reference the issue via "Fixes #1234" or "Refs" in the commit message
+* Run Rubocop prior sending your PR, check Coding Standards in the guide
```

Comment:
I'd remove this

Reply:
I am fine removing Rubocop to keep the list clean